pkg/master/controller.go

```diff
@@ -60,6 +60,7 @@ type Controller struct {
 	ServiceIP         net.IP
 	ServicePort       int
 	PublicServicePort int
+	ServiceEnableDNS  bool
 
```

Comment:
Agreed

On Oct 2, 2015, at 3:05 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/master/controller.go
<https://github.com/kubernetes/kubernetes/pull/14871#discussion_r41056334>:

> @@ -60,6 +60,7 @@ type Controller struct {
>  	ServiceIP         net.IP
>  	ServicePort       int
>  	PublicServicePort int
> +	ServiceEnableDNS  bool
>

Yeah, OK - a more generic factoring would be better - calling it DNS
specifically implies that the master is a DNS server.
… <#>
On Fri, Oct 2, 2015 at 12:01 PM, Andrew Butcher <notifications@github.com>
wrote: In pkg/master/controller.go <
https://github.com/kubernetes/kubernetes/pull/14871#discussion_r41055913> :
> @@ -60,6 +60,7 @@ type Controller struct { > ServiceIP net.IP >
ServicePort int > PublicServicePort int > + ServiceEnableDNS bool > I agree
- I've got that worked out for the most part and will submit soon — Reply
to this email directly or view it on GitHub <
https://github.com/kubernetes/kubernetes/pull/14871/files#r41055913>.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/14871/files#r41056334>.